uaclient/entitlements/__init__.py

```diff
@@ -274,3 +272,12 @@ def check_entitlement_apt_directives_are_unique(
                 )
 
     return True
+
+
+def get_title(cfg: UAConfig, ent_name: str, variant=""):
+    try:
+        return entitlement_factory(cfg, ent_name, variant=variant)(
```

Comment:
Just to double check, what is the advantage of this approach ? We are now creating the entitlement object to fetch the title, which is not necessary.

Is that because of we might miss a variant title by using `ENTITLEMENT_NAME_TO_TITLE` ?

Reply:
It is unfortunately necessary for cis, because the title depends on `self.called_name`

Reply:
I cry so much everytime this kind of comment pops out somewhere
We need to kill this horrible bug